packages/dropdown/src/react/__stories__/index.story.js

```diff
@@ -8,7 +8,7 @@ import * as Icon from '@pluralsight/ps-design-system-icon'
 
 import Dropdown from '../index.js'
 
-const Empty = () => null
+const Empty = React.forwardRef((props, forwardedRef) => <span />)
```

Comment:
In our propTypes we have `{ menu: PropTypes.element.isRequired }` so I modified this to represent that and also portal really does require and element.

Reply:
i'm surprised you hit an error. i've alway though `proptypes.element` just needed any react element so i would have thought a component that returned null would still be valid

Reply:
did the functionality of proptypes change and we upgrade the package recently and that's where the error is coming from?

Reply:
I think it will take a react element that return null but createPortal will not.